### ISSUE TITLE
Where are the GNU APL communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/gnu-apl#27]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is GNU APL supported by one or more large organizations?
* Does GNU APL have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about GNU APL? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for GNU APL? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in GNU APL? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in GNU APL?
* Are there popular newsletters for GNU APL?
* Is GNU APL taught at programming bootcamps? (If so, what are some examples?)
* Is GNU APL taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about GNU APL?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by marnen** [comment]
As I understand it, the language is not very popular these days, and from what I’ve read, the community was small even when it *was* popular. The GNU APL interpreter project seems to have a somewhat active mailing list...

I’ve never really been in the “mainstream” APL community myself; I learned the language more or less on a lark.

Also worth knowing: I chose GNU APL for this track because the interpreter is OSS and seems to stick fairly close to the ISO standard. Dyalog APL is a commercial interpreter with a lot of extra useful features, and it may have more of a community. 

AFAIK, interest in APL remains especially in the financial field, but many former APL developers seem to have moved to its descendants, such as A+, J, K, or Q. 

I’ll try to dig up actual references when I’m not posting on my phone; this is a summary of what I remember from previous research.

**Comment by marnen** [comment]
Here are some links.

GNU APL mailing list: https://lists.gnu.org/mailman/listinfo/bug-apl
APL wiki (with some community listings): https://aplwiki.com/
Dyalog APL forum: https://forums.dyalog.com/

Do these answer your questions? Shall I dig up more info? (And yes, I know I haven't really done much work on this track lately...)

Also worth knowing: repl.it [supports APL](https://repl.it/languages/apl).
